_guides/basics/getting-started/custom-res.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: wiki
+title: Add 60Hz Refresh Rate
+description: "Adds 60Hz to monitors/laptops with no 60Hz refresh rate"
+parent: getting-started
+grand parent: basics
```

Comment:
``parent: Getting Started``
``grand-parent: Basics``

Have to make changes for fields with this error.